Description: Remove unused variables.
 This patch removes the unused variables nexttime and old_time_struct to avoid
 a warning when -Wunused-but-set-variable is passed.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-19

--- a/wmtime/wmtime.c
+++ b/wmtime/wmtime.c
@@ -231,11 +231,9 @@
 	int			but_stat = -1;
 
 	struct tm	*time_struct;
-	struct tm	old_time_struct;
 
 	long		starttime;
 	long		curtime;
-	long		nexttime;
 
 	char		*conffile = NULL;
 
@@ -273,7 +271,6 @@
 	AddMouseRegion(1, 5, 5, 58, 46);
 
 	starttime = time(0);
-	nexttime = starttime + 1;
 
 	curtime = time(0);
 	time_struct = localtime(&curtime);
@@ -283,7 +280,6 @@
 
 		waitpid(0, NULL, WNOHANG);
 		
-		old_time_struct = *time_struct;
 		time_struct = localtime(&curtime);
 
 
